config/jobs/kubernetes/sig-node/node-docker.yaml

```diff
@@ -28,6 +28,43 @@ periodics:
   annotations:
     testgrid-dashboards: sig-node-docker
     testgrid-tab-name: docker-node-conformance
+- name: ci-docker-node-conformance-kubetest2
```

Comment:
I'm curiuos if the provider=gce needs to be somehow specified or it is default?

Reply:
yes, it's by default

Reply:
thank you